src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = WhatIsSex(people, 'm');
+
+  const filterCenturyMan = man.filter((person) => {
+    if (century > 0) {
```

Comment:
>   // avoid using loop and forEach
  // replace `if ()` statement with &&, || or ?:
Please, read the task description carefully

Reply:
Fixed it, lost this block when changed other. I`ll be more careful.